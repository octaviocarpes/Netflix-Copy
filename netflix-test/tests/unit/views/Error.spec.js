import Vue from "vue";
import Error from "@/views/error/Error.vue";

describe("Error.spec.js", () => {
  let cmp, vm;

  beforeEach(() => {
    cmp = Vue.extend(Error);
    vm = new cmp({}).$mount();
  });

  it("has the expected html structure", () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
