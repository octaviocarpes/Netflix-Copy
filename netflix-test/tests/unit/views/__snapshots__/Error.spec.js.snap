// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error.spec.js has the expected html structure 1`] = `
<h1>
  Error
</h1>
`;
